vic/drivers/cesm/src/cesm_put_data.c

```diff
@@ -288,8 +294,8 @@ vic_cesm_put_data()
             }
         }
 
-        if (fabs(1 - AreaFactorSum) > 1e-5) {  // TODO: replace with EPS
-            log_err("AreaFactorSum is not 1 in cesm_put_data.c");
+        if (fabs(1 - AreaFactorSum) > DBL_EPSILON) {
```

Comment:
Don't we have a function in VIC to check that two values are equal to within machine precision?